runtime/manta/src/fee.rs

```diff
@@ -16,10 +16,10 @@
 
 use crate::currency;
 use frame_support::weights::{
-    constants::ExtrinsicBaseWeight, WeightToFeeCoefficient, WeightToFeeCoefficients,
-    WeightToFeePolynomial,
+    WeightToFeeCoefficient, WeightToFeeCoefficients, WeightToFeePolynomial,
 };
 use manta_primitives::types::Balance;
+use runtime_common::ExtrinsicBaseWeight;
```

Comment:
i think we should have these tests below too

https://github.com//blob/1cca061ede75a4c693a3b71bd1b61d7275fe29e4/frame/support/src/weights/extrinsic_weights.rs#L60

And then another assert if our numbers are outside of 80-120% of theirs.

Reply:
Ill add an equals assert, but not sure what's the point of the 80-120 range. We can make a judgement when the assert_eq fails.

Reply:
The point is that we don't want to have to change on every small random variance difference from upstream rerunning the benchmark, which they likely do every release. We only want to notice larger changes.
Could also be 90-110, i just don't want to investigate `assert_eq!(4_000_000,4_000_001)` every time we upgrade deps

Reply:
They don't seem to change often. This specific one `ExtrinsicBaseWeight` didn't change for 9 months.